src/Model/FileVersion.cs

```diff
@@ -82,16 +57,8 @@ public class FileVersion : IToString
         /// <returns>String presentation of the object</returns>
         public override string ToString()
         {
-            return _srcString ?? SerializationHelper.Serialize(this);
+            return SerializationHelper.Serialize(this);
```

Comment:
suspicious change here 🤨

Reply:
Member with name "_srcString" is in base class.  To avoid conflicts I simplified this